containers/docker-compose-unstable.yml

```diff
@@ -19,7 +19,6 @@ services:
     ports:
       - "127.0.0.1:5001:4000"
     build: ..
-    command: /bin/bash -c "sleep 5 && /usr/local/bundle/bin/passenger start -p 4000"
```

Comment:
Wow, so this was the issue! This explains why it failed, CMD from Dockerfile wasn't actually used but `command` from docker-compose was used instead! This means we'll need to also fix `containers/docker-compose-[stable|production].yml`

Reply:
Yes, this was overwriting the "original" CMD from Dockerfile.